src/js/samples/components/Create/CreateSample.tsx

```diff
@@ -219,7 +219,7 @@ export default function CreateSample() {
                     values,
                 }: {
                     errors: FormikErrors<formValues>;
-                    setFieldValue: (field: string, value: string) => void;
+                    setFieldValue: (field: string, value: string | string[] | number[]) => void;
```

Comment:
```suggestion
                    setFieldValue: (field: string, value:  string[] | number[]) => void;
```
Why allow a single string on it's own?

Reply:
Some of the other form values like `group` use strings